GitUI/Script/ScriptRunner.cs

```diff
@@ -152,7 +152,7 @@ string optionDependingOnSelectedRevision
                 }
                 else
                 {
-                    new Executable(command, module.WorkingDir).Start(argument);
+                    new Executable(command, module.WorkingDir).Start(argument ?? string.Empty);
```

Comment:
Is this the right thing to do?
ArgumentString seem to require an argument (also if not provided), so throwing is less worse, but maybe a comment?

Reply:
I found it by creating the following scripts, which is totally valid:
![image](https://user-images.githubusercontent.com/4403806/83727995-406f7c80-a689-11ea-871e-a0b270564e69.png)

I can add a comment

Reply:
Then all scripts have defined an argument, even if it is null. It is not possible to run the scripts without the null argument exists.
If the script parses the empty argument, then it must be adjusted - there are no such scripts now.